linetools/scripts/lt_radec.py

```diff
@@ -43,10 +43,15 @@ def main(args=None):
     coord = ltu.radec_to_coord(icoord)
 
     # Time to print
+    print('===================================')
     print('J{:s}{:s}'.format(coord.ra.to_string(unit=u.hour,sep='',pad=True),
                              coord.dec.to_string(sep='',pad=True,alwayssign=True)))
+    print('   ')
+    print('   {:s} {:s}'.format(coord.ra.to_string(unit=u.hour,sep=':',pad=True),
+                             coord.dec.to_string(sep=':',pad=True,alwayssign=True)))
```

Comment:
Say explicitly that these are J2000 coordinates somewhere?

Reply:
added.  good suggestion.